[Bug]: zomato-clone server crashes on Linux - foodController requires ../models/foodModel but file is foodmodel.js

### Project Name / Day Number

zomato-clone (backend), `public/zomato-clone/server`

### Bug Description

`controllers/foodController.js` imports the model with the wrong filename casing:

```js
const Food = require("../models/foodModel");
```

The actual tracked file is `models/foodmodel.js` (lowercase `m`). On Windows and default macOS the filesystem is case-insensitive, so this resolves and the API works. On a case-sensitive filesystem (Linux, which is the typical deploy target), `require("../models/foodModel")` throws `Error: Cannot find module '../models/foodModel'`.

Because `server.js` mounts `foodRoutes`, which requires `foodController`, which requires the model, this crashes the server at startup on Linux. The entire food CRUD API fails to load. (The model file itself was correctly populated in #7810; only the import casing is wrong.)

### Steps to Reproduce

1. On a case-sensitive filesystem (any Linux host, or a Vercel/Render deploy), run `npm install` then `npm start` in `public/zomato-clone/server`.
2. The server exits with `Error: Cannot find module '../models/foodModel'`.
3. Confirm the mismatch: `git ls-files` shows `public/zomato-clone/server/models/foodmodel.js`, while `controllers/foodController.js` line 1 requires `../models/foodModel`.

### Expected Behavior

The import casing should match the actual filename so the server starts and the food API works on every platform, not just case-insensitive ones.

### Environment

Case-sensitive filesystems (Linux deployments, CI runners). Works by accident on Windows/macOS.

### Checklist

- [x] I have checked that a similar issue does not already exist.
- [x] I have verified this issue on the latest main branch.

--- a/public/zomato-clone/server/controllers/foodController.js
+++ b/public/zomato-clone/server/controllers/foodController.js
@@ -1,4 +1,4 @@
-const Food = require("../models/foodModel");
+const Food = require("../models/foodmodel");
 
 exports.createFood = async (req, res) => {
   const data = await Food.create(req.body);

